src/datasource.ts

```diff
@@ -538,12 +534,14 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
     // Gradually migrate queries to the backend in this condition
     if (
       request.targets.every(
-        target =>
+        (target) =>
           target.metrics?.every(
-            metric =>
+            (metric) =>
               metric.type === 'raw_data' ||
               metric.type === 'raw_document' ||
-              (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene)
+              (request.app === CoreApp.Explore &&
+                target.queryType === QueryType.Lucene &&
+                target.luceneQueryType !== LuceneQueryType.Traces)
```

Comment:
sorry the git diff here is a lot because my linter ran or something, but this is the only line I really intended to change.

Reply:
can it be `target.luceneQueryType === LuceneQueryType.Metric` instead to make it explicit what is being sent to the backend.

Reply:
ooh nice call

Reply:
lol so I tried this and broke a bunch of tests. Not entirely sure if this reflecting issues with tests or code (I'm guessing the second) but I'm going to just merge as is for now, hopefully this code is pretty temporary anyway